Polyhedron/demo/Polyhedron/Viewer.cpp

```diff
@@ -2030,6 +2030,7 @@ void Viewer::scaleScene()
   CGAL::qglviewer::Vec vmin(((float)bbox.xmin()+offset().x)*d->scaler.x(), ((float)bbox.ymin()+offset().y)*d->scaler.y(), ((float)bbox.zmin()+offset().z)*d->scaler.z()),
       vmax(((float)bbox.xmax()+offset().x)*d->scaler.x(), ((float)bbox.ymax()+offset().y)*d->scaler.y(), ((float)bbox.zmax()+offset().z)*d->scaler.z());
   camera()->setPivotPoint((vmin+vmax)*0.5);
+  camera()->setSceneBoundingBox(vmin, vmax);
   camera()->fitBoundingBox(vmin, vmax);
```

Comment:
What is the effects of `camera()->setSceneBoundingBox(vmin, vmax);` and `camera()->fitBoundingBox(vmin, vmax);`?

Reply:
`fitBoundingBox()` will set the location and orientation of the camera so that you can see the box, `setSceneBoundingBox()` will set the camera's frustum so that it won't clip the given box.